.circleci/config.yml

```diff
@@ -362,6 +367,26 @@ workflows:
           requires:
             - prep-build-ts-migration-dashboard
 
+  locales_only:
```

Comment:
This workflow is missing an `all-tests-pass` step, which is what we use as a PR status check to block merging.

Reply:
Thanks, this is good to know!